ts41/utilities.d.ts

```diff
@@ -0,0 +1,8 @@
+/**
```

Comment:
This isn't really a "utility" as they are used in source/utilities.d.ts. It's more of a separate type – and as such deserves a separate file, description, use case and needs to be included in readme.